Composer/packages/server/src/locales/en-US.json

```diff
@@ -404,6 +425,9 @@
   "chris_whitten_11df1f35": {
     "message": "Chris Whitten"
```

Comment:
Not part of this PR, but fun to localize - K’ris Uit’en

Reply:
lol